packages/wrangler/templates/startDevWorker/ProxyWorker.ts

```diff
@@ -250,3 +255,20 @@ function insertLiveReloadScript(
 
 	return htmlRewriter.transform(response);
 }
+
+/**
+ * Rewrite references to URLs in request headers.
+ *
+ * This function is used to map the URLs in headers like Origin and Access-Control-Allow-Origin
```

Comment:
Access-Control-Allow-Origin is a response header so I assume you expect this function to be used in both directions?

Reply:
I'll tweak the description.